tests/Assets/SimpleEntityWithEnum.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineTest\Laminas\Hydrator\Assets;
+
+use const PHP_VERSION_ID;
+
+if (PHP_VERSION_ID >= 80100) {
```

Comment:
We only use this asset when on PHP 8.1+ - IMO declaring it conditionally could be worse :thinking: